src/commands/utils/color.ts

```diff
@@ -16,6 +18,9 @@ export const colorCommand: SlashCommand = async (i) => {
 	} else if (hexRegex.test("#" + input)) {
 		fullString = "#" + input;
 		hexOnly = input;
+	} else if (Object.prototype.hasOwnProperty.call(cssColorKeywords as { [key: string]: string }, input)) {
```

Comment:
NO

Reply:
> NO

whats the issue here, also i updated it a bit

Reply:
Does `input in cssColorKeywords` not work

Reply:
> Does `input in cssColorKeywords` not work

oops